xilinx/vc707/constraints/vc707-master.xdc

```diff
@@ -1,6 +1,3 @@
 #-------------- MCS Generation ----------------------
 set_property CFGBVS GND                               [current_design]
 set_property CONFIG_VOLTAGE 1.8                       [current_design]
-
-set_property EXTRACT_ENABLE YES                       [get_cells dut_/spi_0_1/mac/phy/txd_reg*]
```

Comment:
Can you explain to me why you made this change?

Reply:
No technical reason. 
When I made the pull request https://github.com/sifive/freedom/pull/118 I just used where the freedom was pointing at fpga-shells and the commit included the patch for  `Revert "SPI speedup related FPGA changes"` which changing the `set_property EXTRACT_ENABLE YES`.
I am fine being merged without this change.

Reply:
If you want me to regenerate the pull request without the https://github.com/sifive/fpga-shells/commit/a0cc85e01d86bf7a8cd0a312b70da5db12e5d5b9, please let me know.